### ISSUE TITLE
collect respondent's parents' phone numbers?

### ISSUE THREAD

**Issue by ekressmiller** [mplp/docassemble-MLHPPOAndProposedOrder#214]
In LHI, we don't collect respondent's parents' phone numbers, and we just put their addresses on the order forms:
![image](https://github.com/user-attachments/assets/15e9f7b2-b646-4eb3-8355-928664c4c044)

Continue that, or start collecting phone like we currently have in Docassemble.

Also note, the parent info seems to be missing from the sexual assault order in LHI, though it was collected, is that intentional?
![image](https://github.com/user-attachments/assets/99a12158-a199-43d9-8b94-899bb8824f85)

**Comment by ekressmiller** [comment]
@ABillmire @normon66 

Please see the info above. In the Docassemble version, we currently _do_ collect the phone number for respondents' parent(s) when appropriate, and add them to the forms. (I'm guessing we built that in because the forms refer to the phone number, and we hadn't yet checked that detail in LHI.)  

However, I just noticed in the LHI version we _don't_ collect the parents' phone numbers. Would it be better to roll back and collect **only** the parental address, not phone? Or is it OK to optionally collect their phone?

Also, note the second finding above, is parental info intentionally left off the sexual assault petition or is that a bug in the LHI version? (or a misunderstanding by me)

**Comment by ABillmire** [comment]
Let's optionally collect the parents' phone numbers. A phone number is not required in the court rules.

Re: the name(s) and addresse(s) of the minor respondent's parents/guardians/custodians, please add those to the minor PPOs if the info is provided. Let's make those optional fields for the names and addresses of parents/guardians/custodians as well. I want to document the reason: MCR 3.706(D) says that "If the respondent is a minor, **and the whereabouts of the respondent's parent or parents, guardian, or custodian is known**, the petitioner shall also in the same manner serve the order on the respondent's parent or parents, guardian, or custodian."

Likewise MCR 3.703(C) provides that a **petition** against a minor must list the names and addresses of the minor respondent's parent or parents, guardian, or custodian **if known or can be easily ascertained**. I'm going to ask SCAO if the petition forms should be revised for that, because currently there are no spaces for that info.

I'd also like to make a wording change in the interview:

-Where it says "Because [respondent] is an unemancipated minor, both [respondent] and his parents, guardian(s), or custodian(s) must be served with your petition for a personal protection order," I'd like to change it to "If you know or can easily find out the names and addresses of [respondent]'s parent(s), guardian(s), or custodian(s), fill in the information on the following pages." (or something to that effect)

**Comment by ekressmiller** [comment]
Got it, thanks Amy!

And just double-checking that all this applies to the minor sexual assault petition as well? (see above note about discrepancy in LHI version)

**Comment by ABillmire** [comment]
Yes. I believe that discrepancy was unintentional. I can't find any reason for it.

**Comment by normon66** [comment]
Good catch, @ekressmiller! The discrepancy in the sexual assault order was indeed unintentional. You can see in the image below that those fields were missing, and probably have been all along. All other minor-respondent orders, including the Spanish versions (which were just marked up English forms), included the address of the parents or guardians.
   ![image](https://github.com/user-attachments/assets/024d0569-9df6-45fd-9fc5-d64d6b7c42fe)

The minor sexual assault order will now include that information, and I'll also update the interview and all minor respondent orders to optionally include phone numbers for parents and guardians, as well.

**Comment by ekressmiller** [comment]
Thanks @normon66! I defer to @ABillmire, but I don't think you need to update the LHI interview with that phone number change since we're getting closer all the time to being ready with the Docassemble version.

**Comment by normon66** [comment]
@ABillmire - sounds good. Would you still like me to make the names and addresses for parents optional?

**Comment by ekressmiller** [comment]
@normon66 again I defer to @ABillmire but my two cents is that change is sort of an enhancement/not necessary in LHI.

**Comment by ABillmire** [comment]
@normon66 If it is a quick change, then yes go ahead please.

**Comment by ekressmiller** [comment]
@normon66 Just to tack on to Amy's go-ahead, we just discussed it and she said that any information is better than none. So if they have an address but no name for a parent, for example, we would want to include the address.

**Comment by normon66** [comment]
The live Petition for PPO interview has been updated to allow for optional parent names and contact information.

**Comment by ekressmiller** [comment]
@ABillmire just tagging you so you can take a look at this again
